macros/anaUltraThreshold.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python
+#!/bin/env python2.7
```

Comment:
Yes initialization of dict_vfatTrimMaskData at line 183 requires dict comprehension which was added in python2.7.  Does this implementation pose a problem?